### PR TITLE
Temporary fix for logging to console output

### PR DESCRIPTION
**Pull request by tamimfatahi**
PR #614 had a logging to console output issue. I added in a TODO request in `inputs.py` and will create an issue for later on improving how we log from multiple modules as the current format isn't enough to support that from my understanding. On the console output end for what users read during EXOTIC use, everything should be fine at least. Also, caught a bug regarding the name change from `Uncertainity` -> `Uncertainty` in the `inits.json` file.

Edit: Forgot to add, solves issue #618

### CODE REVIEW

File: exotic/inputs.py [hunk 1]
```diff
@@ -1,13 +1,20 @@
...
     from .util import user_input, dms_to_dd, open_elevation, typecast_check, init_params
 
-
+# TODO Need to revamp logging file handler in the future for all of EXOTIC modules. Not correct formatting.
 log = logging.getLogger(__name__)
+consoleFormatter = logging.Formatter("%(message)s")
+consoleHandler = logging.StreamHandler(sys.stdout)
+consoleHandler.setFormatter(consoleFormatter)
+consoleHandler.setLevel(logging.INFO)
+log.addHandler(consoleHandler)
```
Comment (jpl-jengelke) [inline exotic/inputs.py:17]: I see a console handler. Is there another handler for files?

**Comment by jpl-jengelke** [comment]
Here is a good formatter that I have used in the past:
```
%(levelname)s:%(filename)s-%(funcName)s:%(msg)s
```
alternately
```
%(levelname)s:%(filename)s-%(funcName)s-%(lineno)s:%(msg)s
```

**Comment by rzellem** [comment]
Looks like @jpl-jengelke has some recommendations on how to update the code; I will wait until @tamimfatahi has addressed his comments/suggestions before pulling into develop - no rush!

**Comment by tamimfatahi** [comment]
exotic.log example: 

`2021-04-21T09:25:11.996 [MainThread  ] INFO   main:1684 - 
**************************
INFO:__main__:
**************************
2021-04-21T09:25:11.996 [MainThread  ] INFO   main:1685 - Complete Reduction Routine
INFO:__main__:Complete Reduction Routine
2021-04-21T09:25:11.996 [MainThread  ] INFO   main:1686 - **************************
INFO:__main__:**************************
INFO:inputs:
Your current working directory is: /Users/abdullahfatahi/Documents/ExoplanetWatch/EXOTIC
INFO:inputs:Potential initialization files I've found in /Users/abdullahfatahi/Documents/ExoplanetWatch/EXOTIC are: 
INFO:inputs:	/Users/abdullahfatahi/Documents/ExoplanetWatch/EXOTIC/eaConf.json
INFO:inputs:	/Users/abdullahfatahi/Documents/ExoplanetWatch/EXOTIC/inits.json
WARNING:inputs:Error: The directory entered does not exist. Please try again. Make sure to follow this 
formatting (using whichever directory you choose): /sample-data/results`

You'll notice `inputs` is from `inputs.py`

**Pull request merged**